docs/user-guide/management/aggregation-cache.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Streaming Aggregation in OpenObserv
+description: Learn how Streaming Aggregation accelerates dashboards using aggregation cache in OpenObserve Enterprise.
+---
+This document helps you understand what Streaming Aggregation is and how it relates to aggregation cache in OpenObserve.
+
+## What is Streaming Aggregation
+Streaming Aggregation is an Enterprise setting in OpenObserve that enables aggregation cache.
+
+!!! note "Who can use it"
+    All Enterprise users. 
+
+!!! note "Where to find it"
+    To enable aggregation cache: 
+
+    1. Go to **Management > General Settings**.  
+    2. Turn on the **Enable Streaming Aggregation** toggle.
+    3. Select **Save**.
+
+## What is aggregation cache
+Aggregation cache improves performance by storing the results of each time partition the first time a query runs. This applies to dashboards, log queries, or any other place where aggregate SQL queries are used. When you run a query again with the same stream and filters, and for the same or overlapping time range, OpenObserve reuses the stored partitions and computes only the missing ones. The final result remains accurate because OpenObserve merges the stored and new partitions using safe operations such as adding sums and counts before computing averages.
+
+## How aggregation cache works
+This section explains what happens at query time. Aggregation cache stores mergeable results per time partition on the first run, then reuses matching partitions on later runs that use the same stream, filters, and grouping over the same or overlapping time range.
+
```

Comment:
I think here we need to specify intermediate record batches 
with example like to if sql has avg 
- avg=sum/count
- the intermediate record batches sum and count is stored for every partition 
- in the end these are aggregated along with the data for the missing partitions
- thus honoring the accuracy of the query